theano/gpuarray/extra_ops.py

```diff
@@ -468,4 +469,4 @@ def use_gpu_cumsumop(op, ctx_name, inputs, outputs):
         if axis is None:
             axis = 0
         assert isinstance(x.type, GpuArrayType)
-        return GpuCumsum(axis)(x)
+        return GpuCumsum(axis, ctx_name)(x)
```

Comment:
As documented [here](http://deeplearning.net/software/theano_versions/dev/extending/extending_theano_gpu.html), the best practice is to infer the context during `make_node`, not to force the context when creating the Op.
Please remove the `context_name` attribute.